MetasysServices/Models/MetasysObject.cs

```diff
@@ -187,11 +188,19 @@ internal MetasysObject(JToken token, ApiVersion version, IEnumerable<MetasysObje
             }
             try
             {
-                Classification = jobj.ContainsKey("classification") ? token["classification"].Value<string>() : null;
+                if (jobj.ContainsKey("classification")) 
+                {
+                    var tokenValue = token["classification"].Value<string>().Trim();
+                    Classification = tokenValue == string.Empty ? DefaultClassification : tokenValue;
+                }
+                else
+                {
+                    Classification = DefaultClassification;
```

Comment:
Actually I'm fine leaving Classification as `null` if the property doesn't exist on the node. It should never happen anyway.